1000-SR.1/T-SP-SR.1-2012-MSW-E.adoc

```diff
@@ -0,0 +1,519 @@
+*_Kenya_* +
+The international telegram messenger delivery services to physical addresses are not accepted.
+
+a|*_Koweït_* +
+*_Kuwait_* +
+*_Kuwait_* +
+Incoming, outgoing and transit telegrams from or to Israel are not accepted.
+
+a|*_Liban_* +
+*_Lebanon_* +
+*_Líbano_* +
+Incoming, outgoing and transit telegrams from or to Israel are not accepted.
+
+a|*_Luxembourg_* +
+*_Luxembourg_* +
+*_Luxemburgo_* +
+The telex service is no longer provided. +
+The telegram service is no longer provided +
+EPT announces cancellation of the operator service.
+
+a|*_Madagascar_* +
+*_Madagascar_* +
+*_Madagascar_* +
+The telegram service is no longer provided
+
+a|*_Malaisie_* +
+*_Malaysia_* +
+*_Malasia_* +
+The telex service is no longer provided.
+
+a|*_Malawi_* +
+*_Malawi_* +
+*_Malawi_* +
+Collect calls to mobile numbers are not accepted
+
+a|*_Maldives_* +
+*_Maldives_* +
+*_Maldivas_* +
+Collect calls for both outgoing and incoming  are not accepted
+
+a|*_Malte_* +
+*_Malta_* +
+*_Malta_* +
+The telegram service is no longer provided +
+The telex service is no longer provided.
+
+a|*_Maurice_* +
+*_Mauritius_* +
+*_Mauricio_* +
+Letter telegrams (LT) and government letter telegrams (LTF) are no longer admitted to and from Mauritius.
+
+a|*_Maroc_* +
+*_Morocco_* +
+*_Marruecos_* +
+Collect calls to mobile numbers and payphones are not accepted
+
+a|*_Mozambique_* +
+*_Mozambique_* +
+*_Mozambique_* +
+The telex service is no longer provided.
+
+a|*_Namibie_* +
+*_Namibia_* +
+*_Namibia_* +
+Collect calls to mobile numbers  are not accepted
+
+a|*_Pays-Bas_* +
+*_Netherlands_* +
+*_Países Bajos_* +
+Collect calls to mobile numbers  are not accepted
+
+a|*_Nouvelle-Calédonie_* +
+*_New Caledonia_* +
+*_Nueva Caledonia_* +
+The telex service is no longer provided. +
+The telegram service is no longer provided +
+Collect calls for both outgoing and incoming  are not accepted
+
+a|*_Nouvelle-Zélande_* +
+*_New Zealand_* +
+*_Nueva Zelandia_* +
+The telegram service is no longer provided
+
+a|*_Nigéria_* +
+*_Nigeria_* +
+*_Nigeria_* +
+Due to present congestion in Lagos Port, the harbour authority will reject messages from any ship without the following particulars: a) ship's name; b) call sign; c) agent's name; d) overall length; e) draught; f) tonnage loaded; g) nature of cargo; h) last port of call; i) date and local time of ship arrival outside bar.
+
+a|*_Norvège_* +
+*_Norway_* +
+*_Noruega_* +
+Telenor will no longer provide collect calls from Norway. Collect calls to Norway will however continue to be accepted.
+
+a|*_Pakistan_* +
+*_Pakistan_* +
+*_Pakistán_* +
+The telecommunication service with Israel is suspended. +
+The telex service is no longer provided. +
+The telegram service is no longer provided.
+
+a|*_Panama_* +
+*_Panama_* +
+*_Panamá_* +
+Collect calls to mobile numbers  are not accepted
+
+a|*_Papouasie-Nouvelle-Guinée_* +
+*_Papua New Guinea_* +
+*_Papua Nueva Guinea_* +
+Collect calls to mobile numbers  are not accepted +
```

Comment:
Double spaces should be made single.

Reply:
Excuse me @ronaldtse , I don't see any double spaces. Could you please indicate me where are they?